checkov/common/output/common.py

```diff
@@ -52,3 +53,9 @@ def format_string_to_licenses(licenses_str: str) -> list[str]:
 def compare_table_items_severity(table_item: dict[str, str]) -> int:
     severity = (table_item.get("severity") or DEFAULT_SEVERITY).upper()
     return Severities[severity].level
+
+
+def validate_lines(lines: list[int] | None) -> list[int] | None:
+    if lines and len(lines) > 0 and lines[0] > 0 and lines[1] > 0:
```

Comment:
```suggestion
    if lines and lines[0] > 0 and lines[1] > 0:
```
if the length is 0 the first condition will be false